### ISSUE TITLE
[BUG]  Can not install latest version of integration

### ISSUE THREAD

**Issue by dwdonnelly** [romedtino/simple-wyze-vac#90]
**Describe the bug**
Attempted to install new version - 1.9.2.  - but no option to install.

**To Reproduce**
No steps to install the latest version.

**Expected behavior**
A method to install the latest version.

**Screenshots**
N/A

**Simple Wyze Vac Version**
- 1.9.1

**Wyze Vacuum Firmware Version**
- N/A

**Home Assistant Version**
![image](https://github.com/user-attachments/assets/23dd2478-3ea4-4c7e-aa01-8cb6473b6441)

**Additional context**
None

**Comment by dwdonnelly** [comment]
@romedtino   Any thoughts on this issue?  Actions to try?

**Comment by romedtino** [comment]
There's no problem with the the latest version of HA and installing this integration. Please follow HACS guidance on adding custom integrations.

**Comment by dwdonnelly** [comment]
@romedtino My apologies - but I must be doing something wrong.  I have gone to HACS, read the documentation:  selected update the information, redownloaded the integration, and followed the prompts to install/update the integration....HA restarted.  The integration is still showing at 1.9.1.

I followed this guidance:
![image](https://github.com/user-attachments/assets/4d4bb962-7566-45d9-948e-af78cb509c16)

Settings -> updates is blank.

I did option 2. All the way through restarting HA.  Still at 1.9.1.

What am I missing?  I too am learning.
